csfieldguide/static/interactives/lzss-compression/scss/lzss-compression.scss

```diff
@@ -0,0 +1,118 @@
+html{
+	font-family: "Roboto",sans-serif;
+}
+
+body {
+	margin: 0px;
+}
+
+h1 {
+	text-align: center;
+}
+
+input {
+	text-align: center;
+	font: 1rem "Courier New", monospace;
+}
+
+
+#interactive-lzss-compression {
+	display: flex;
```

Comment:
Same with bootstrap classes in this file :) Also same with the bootstrap buttons